src/shared/components/VideoTileBase/VideoTileBase.styles.tsx

```diff
@@ -62,9 +62,13 @@ export const TextContainer = styled.div`
 export const KebabMenuButtonIcon = styled(IconButton)<ActiveProps>`
   ${square(32)};
 
+  opacity: 1;
   margin-left: ${sizes(2)};
-  opacity: ${({ isActive }) => (isActive ? 1 : 0)};
   pointer-events: ${({ isActive }) => (isActive ? 'auto' : 'none')};
+
+  ${media.md} {
+    opacity: ${({ isActive }) => (isActive ? 1 : 0)};
+  }
```

Comment:
Let's not base this on screen width but instead of media query which tells us whether the device has a pointer. `@media (any-pointer: none)` should do the trick